homeassistant/components/fyta/sensor.py

```diff
@@ -0,0 +1,179 @@
+"""Summary data from Fyta."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Final
+
+from fyta_cli.fyta_connector import PLANT_STATUS
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE, EntityCategory, UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import FytaCoordinator
+from .entity import FytaCoordinatorEntity, FytaPlantEntity
+
+
+@dataclass(frozen=True)
+class FytaSensorEntityDescription(SensorEntityDescription):
+    """Describes Fyta sensor entity."""
+
+    value_fn: Callable[[str | int | float], str | int | float | datetime] = (
+        lambda value: value
+    )
+
+
+PLANT_STATUS_LIST: list[str] = ["too_low", "low", "perfect", "high", "too_high"]
+
+SENSORS: Final[list[FytaSensorEntityDescription]] = [
+    FytaSensorEntityDescription(
+        key="plant_name",
+        translation_key="plant_name",
+    ),
+    FytaSensorEntityDescription(
+        key="scientific_name",
+        translation_key="scientific_name",
+    ),
+    FytaSensorEntityDescription(
+        key="status",
+        translation_key="plant_status",
+        device_class=SensorDeviceClass.ENUM,
+        options=PLANT_STATUS_LIST,
+    ),
```

Comment:
```suggestion
        options=PLANT_STATUS_LIST,
        value_fn=lambda value: PLANT_STATUS[value],
    ),
```

Reply:
Thank you. Integrated this for all status entities.